lib/spack/spack/test/installer.py

```diff
@@ -162,6 +166,25 @@ def _spec(spec):
     assert 'Installing a from binary cache' in capfd.readouterr()[0]
 
 
+def test_try_install_from_binary_cache(install_mockery, mock_packages,
+                                       monkeypatch, capsys):
+    """Tests to add coverage to _try_install_from_binary_cache."""
```

Comment:
As a minor comment, I would prefer the docstrings (if there's need for any) to reflect the behavior they are testing rather than the function they are adding coverage to. Here, for instance, I feel like the name of the test function is more informative than the docstring.

Reply:
Excellent point.  I was getting sloppy.

Reply:
Fair enough.  I've gone through and modified or removed this and similar docstrings throughout this module.

I've also split test functions when they contained multiple, non-dependent checks.